pkgs/by-name/lu/lubelogger/package.nix

```
@@ -37,7 +37,6 @@ buildDotnetModule rec {
     homepage = "https://lubelogger.com";
     changelog = "https://github.com/hargata/lubelog/releases/tag/v${version}";
     license = licenses.mit;
-    maintainers = with maintainers; [ samasaur ];
```

```suggestion
    maintainers = [ ];
```
and this should probably be a separate PR from the new module